flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/TableSinkITCase.scala

```diff
@@ -76,4 +81,22 @@ class TableSinkITCase extends BatchTestBase {
     Assert.assertEquals(expected.sorted, result2.sorted)
   }
 
+  @Test
+  def testCollectSinkConfiguration(): Unit = {
+    tEnv.getConfig.getConfiguration.set(
+      CollectSinkOperatorFactory.MAX_BATCH_SIZE, MemorySize.parse("1b"))
+    try {
+      checkResult("SELECT 1", Seq(row(1)))
+      Assert.fail("Expecting exception thrown from collect sink")
+    } catch {
+      case e: Exception =>
+        MatcherAssert.assertThat(
+          e,
+          FlinkMatchers.containsMessage("Record size is too large for CollectSinkFunction"))
```

Comment:
Could you also tell users how to change the configuretion and the config name in the exception message?

Reply:
> Could you also tell users how to change the configuretion and the config name in the exception message?

Thank you for above comments，but I still hava a question. Is there a maximum limit on this value? If users can configure it as much as possible, there should be a maximum limit, such as not exceeding the size of taskmanager's taskmanager. memory. process. size？ 
Thank you for your reply！